internal/provider/kubernetes/controller.go

```diff
@@ -294,6 +294,21 @@ func (r *gatewayAPIReconciler) Reconcile(ctx context.Context, _ reconcile.Reques
 		}
 	}
 
+	// Add all ClientTrafficPolicies
+	clientTrafficPolicies := egv1a1.ClientTrafficPolicyList{}
+	if err := r.client.List(ctx, &clientTrafficPolicies); err != nil {
+		return reconcile.Result{}, fmt.Errorf("error listing clienttrafficpolicies: %v", err)
```

Comment:
nit: lowercase is hard reading

Reply:
So you're suggesting to convert clienttrafficpolicies to ClientTrafficPolicies ?

Reply:
yes